### ISSUE TITLE
Example code generates broken text

### ISSUE THREAD

**Issue by dansmith01** [rstudio/shinymeta#94]
I ran the example code from https://rstudio.github.io/shinymeta/index.html, and got this for output:

<img width="531" alt="wonkytext" src="https://user-images.githubusercontent.com/2066273/110546637-9b844300-80e3-11eb-9dda-3498b891b26b.png">

Same result in Chrome and Firefox. Ran update.packages() today on R 4.0.4; windows 10.

**Comment by dansmith01** [comment]
I can get the expected output by changing
```r
output$code <- renderPrint({
  expandChain(output$Summary())
})
```
to
```r
output$code <- renderText({
  formatCode(expandChain(output$Summary()))
})
```

**Comment by jcheng5** [comment]
Fixed via #93. Sorry for the inconvenience!
